teachertool/src/types/criteria.ts

```diff
@@ -20,6 +20,7 @@ export interface CriteriaInstance {
     instanceId: string;
     params: CriteriaParameterValue[] | undefined;
     userFeedback?: UserFeedback;
+    deleted?: boolean;
```

Comment:
I think we should still ensure deleted criteria get filtered out of export/serialization. It's not as big of an issue since it gets removed once the toast goes away, but the user could still export while the toast is around (or more likely, close the browser/tab, so it gets serialized and stored in browser storage), at which point the deleted criteria would be there forever since the delete code won't run on it anymore.